ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SyslogIntegrationForm.tsx

```diff
@@ -197,6 +206,24 @@ function SyslogIntegrationForm({
                                 isDisabled={!isEditable}
                             />
                         </FormLabelGroup>
+                        <FormLabelGroup
+                            isRequired
+                            label="Maximum message size"
+                            fieldId="syslog.maxMessageSize"
+                            touched={touched}
+                            errors={errors}
+                            helperText="A number of bytes between 0 and 1048576 to chunk messages in. 0 means messages will not be chunked."
```

Comment:
nit: The definite article is more like our style of helper text in other forms. Also, starting a sentence with a numeral looks odd.


```suggestion
                            helperText="The number of bytes between 0 and 1048576 to chunk messages in. A 0 means messages will not be chunked."
```